pwa/src/components/paginate/Paginate.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import * as styles from "./Paginate.module.css";
+
+import ReactPaginate from "react-paginate";
+import { faChevronRight, faChevronLeft } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { Button } from "@gemeente-denhaag/components-react";
+
+interface PaginateProps {
+  totalPages: number;
+  currentPage: number;
+  changePage: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const Paginate: React.FC<PaginateProps> = ({ totalPages, currentPage, changePage }) => {
+  if (totalPages < 1) return <></>; // no pages available
+
+  return (
+    <ReactPaginate
+      className={styles.container}
+      disabledClassName={styles.disabled}
+      activeClassName={styles.currentPage}
+      onPageChange={(e: any) => changePage(e.selected + 1)}
+      forcePage={currentPage - 1}
+      pageRangeDisplayed={3}
+      pageCount={totalPages}
+      disableInitialCallback
+      marginPagesDisplayed={2}
+      breakLabel="..."
+      nextLabel={
+        <Button className={styles.button}>
+          <FontAwesomeIcon icon={faChevronRight} />
+        </Button>
+      }
+      previousLabel={
+        <Button className={styles.button}>
+          <FontAwesomeIcon icon={faChevronLeft} />
+        </Button>
+      }
```

Comment:
I think these buttons look a bit wierd.
I would remove the tag and just have it as a string like `< Previous` and `Next >` 
And having it grayed out like so.
![image](https://user-images.githubusercontent.com/43807324/217209298-c25dae6a-21e3-409f-a996-10478b55c41b.png)